Zc-specification/Zc.adoc

```diff
@@ -11,6 +11,7 @@
 [width="100%",options=header]
 |====================================================================================
 |Version  | change
+|v1.0.4-1    | Added rule that Zcf imply F and Zcd imply D - discuss in https://github.com/riscv/riscv-code-size-reduction/issues/221

```

Comment:
what is the reason for adding the -1 suffix? In my understanding of semantic versioning, such a small change would simply increment the patch digit of the version number, i.e. go to 1.0.5

Reply:
Where is this defined? https://semver.org/ Usually uses hyphens for pre-release versions like -alpha/-beta/-rc1...

Reply:
I would have written (beware I am not a native english speaker):
```suggestion
|v1.0.4-1    | Added rule that Zcf implies F and Zcd implies D - discussed in https://github.com/riscv/riscv-code-size-reduction/issues/221

```